Task3_Products/crud.py

```diff
@@ -0,0 +1,222 @@
+from typing import Optional, Literal
```

Comment:
Just for notice, no need to fix: it's better to separate entities in separate files along one module (for example `crud/category.py` and `crud/product.py`). To be more proficient it's better to separate actual interaction with DB (persistent layer) from some business logic (service layer) and endpoints (representation layer), last layer you separated good. Read about 3-layered architecture to learn more